bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -4,7 +4,7 @@
 
 {% extends "base-pebbles.html" %}
 
-{% add_lang_files "mozorg/internet-health/index" %}
+{% add_lang_files "mozorg/internet-health/shared" %}
```

Comment:
It's the same name as the template, there's no need to add the lang file. Can you give it a try locally? It seems to work just fine for me with DEV=False: if the lang file is activated I get the new hub, if it's not I fall back to the old internet-health page